### DISCUSSION TITLE
How to deploy monitor on milvus-server without k8s?

### DISCUSSION THREAD

**Discussion by xiaojuan8** [milvus-io/milvus#15913] [Q&A and General discussion]
Hi, I deployed a milvus cluster, and want to deploy a service monitor for milvus-server, the official wiki tells me to deploy monitor with k8s (https://milvus.io/cn/docs/v2.0.x/monitor.md).But error occurred when I follow the deployment steps as shown below:
![image](https://user-images.githubusercontent.com/74544828/157007526-e506833f-b6cc-470a-ad03-6e89e66ea1f8.png)

This error may show that my k8s version need to update, which is not allowed for other reasons.

So I am wondering if there are any other ways to deploy monitor service for milvus-cluster (if it is possible to deploy monitor service for every node in cluster respectively?)

**Comment by LoveEachDay** [comment]
Could you provide your kubernetes cluster version via the following command:
`kubectl get nodes`
And also check the api-resources use the following command:
`kubectl api-resources | grep CustomResourceDefinition`

If your cluster version is below the requirements, you need to manually change the manifests in the `setup`, change the api version to:
```
apiVersion: apiextensions.k8s.io/<your-system-supported-version>
kind: CustomResourceDefinition
```

**Reply by xiaojuan8** [reply]
Hi, info as follows:
1. k8s version: v1.11.5
2. kubectl api-resources execution error
[root@host10301345 xiaojuan8]# kubectl api-resources | grep CustomResourceDefinition
error: unable to retrieve the complete list of server APIs: metrics.k8s.io/v1beta1: the server is currently unable to handle the request


Thanks for your solution!

**Reply by xiaojuan8** [reply]
I will try to reset config
